style.css

```diff
@@ -108,26 +107,544 @@ a {
+  text-align: center;
+  color: #ebebff;
+  margin: 0 20px 40px 20px;
+}
+
+/* form */
+.form {
+  display: flex;
+  margin: 20px;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.form-text {
+  font-size: 28px;
+  text-align: center;
+  margin: 20px;
+}
+
+input {
+  border-top-style: hidden;
+  border-right-style: hidden;
+  border-left-style: hidden;
+  border-bottom-style: solid;
+}
+
+input[type="text"],
+[type="email"] {
+  width: 100%;
+  padding: 16px 14px;
+  margin: 0 0 10px 0;
+  box-sizing: border-box;
+  border-radius: 4px;
+}
+
+textarea {
+  width: 100%;
+  height: 170px;
+  padding: 12px 8px;
+  border: none;
+  border-radius: 2px;
+  background-color: #fbf8f7;
+  margin: 0 0 10px 0;
+}
+
+textarea:hover::placeholder {
+  background: #d1cbc9;
+}
+
+.btn-1 {
+  padding: 12px;
+  border-radius: 8px;
+  font-weight: 500;
+  font-size: 17px;
+  letter-spacing: 0.03em;
+  background-color: #fff;
+  color: #6070ff;
+  margin: 0 0 20px 0;
+}
+
+.btn-1:active {
+  background: rgba(255, 255, 255, 0.849);
+  font-weight: 600;
+  color: #333;
+}
+
+@media screen and (min-width: 786px) {
+  #icon-menu {
+    display: none;
+  }
+
+  .normal-button ul {
+    display: flex;
+    align-items: center;
+    justify-content: space-evenly;
+    visibility: visible;
+  }
+
+  .normal-button li {
+    margin-left: 30px;
+    font-size: 16px;
+    font-weight: 500;
+    color: #344563;
+  }
+
+  .normal-button a {
+    font-size: 1.2rem;
+    color: #344563;
+  }
+
+  .bg-image-1 {
+    background-image: url("./assets/desktop-header.svg");
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: contain;
+    border: 1px solid transparent;
+  }
+
+  .text-container {
+    max-width: 800px;
+    margin: 120px auto 0 auto;
+  }
+
+  .lg-text-1 {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .headline {
+    padding: 0 0 4% 0;
+  }
+
+  .social-media {
+    display: flex;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 40px;
+    margin: 20px auto 120px auto;
+    max-width: 800px;
+    padding: 0 20px;
+  }
+
+  /* works */
+  .works {
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-row-gap: 120px;
+    padding: 120px 180px;
+  }
+
+  .card-img-left,
+  .card-img-right {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    background-color: #fff;
+    border-radius: 16px;
+  }
+
+  .card-img-right ul {
+    margin: 20px;
+    order: 1;
+  }
+
+  /* .card-img-left ul,
+  .card-img-right ul {
+    margin: 20px;
+    display: flex;
+  } */
+
+  .img-1-1 {
+    display: none;
+  }
+
+  .img-1-2 {
+    display: block;
+  }
+
+  .block-2 {
+    margin: 0 40px 20px 40px;
+  }
+
+  /* about */
+  .about {
+    display: flex;
+    flex-direction: row;
+    align-items: flex-start;
+  }
+
+  .div-container {
+    margin: 0 auto;
+    width: 100%;
+    padding: 10%;
+    gap: 5%;
+  }
```

Comment:
- [ ] Kindly make to adapt your website to the desktop version as you had already set that breakpoint of `min-width: 786px` to be the desktop version. Please make sure to adjust your margin and width to this `about-section` to look exactly as how it looks on the Figma design. :100: 

    - ### Your design (width: 787px)
     ![image](https://user-images.githubusercontent.com/87197412/186937877-473fbb06-381e-4db3-b1ac-44798d2f370b.png)


  - ### Figma design
   ![image](https://user-images.githubusercontent.com/87197412/186938117-818d885f-09ca-48b9-bf0e-f27d049b9740.png)

Reply:
Resolved. Thank you!